code/modules/projectiles/gun_attachables.dm

```diff
@@ -934,10 +934,18 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	scatter_mod = -10
 	movement_acc_penalty_mod = 0.1
 
-/obj/item/attachable/stock/rifle/irremoveable
+/obj/item/attachable/stock/irremoveable/rifle
 	name = "\improper M412 solid stock"
+	icon_state = "riflestock"
+	attach_icon = "riflestock_a"
+	wield_delay_mod = 0.2 SECONDS
+	size_mod = 1
 	pixel_shift_x = 32
 	pixel_shift_y = 13
+	accuracy_mod = 0.05
+	recoil_mod = -3
+	scatter_mod = -10
+	movement_acc_penalty_mod = 0.1
```

Comment:
you don't need those stats, the initial stock for it dindt use to have those, even if it was a typepath of rifle it just had all those as 0, so as a irremoveable parent it doens't need

Reply:
Putting all the stats into the weapon itself when it has a irremovable attachment does make sense. I‘ll remove the stats then, thanks.

Reply:
Removed these, the size mod, and the wield delay in commit. Thanks again.